### PR TITLE
UI Improvements to "Manage" sections of Dashboard

### PR DESCRIPTION
**Pull request by dburka**
Story: https://www.pivotaltracker.com/story/show/171486895

Small clean up all over the place to make the Dashboard more usable. Also added OPD Load numbers to the UI make the Facilities screens make more sense.

### Before

Screenshot 2020-02-26 at 2 35 48 PM

### After

Screenshot 2020-02-26 at 2 35 32 PM

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -67,13 +65,25 @@ h2 {
...
     font-size: 20px;
     color: $grey-darkest;
     font-weight: 700;
     font-family: "Roboto", sans-serif;
+    margin-bottom: 0.5em;
+}
+
+h2 span,
+h3 span {
+    display: block;
```
Comment (prabhanshuguptagit) [inline app/assets/stylesheets/application.scss]: Not sure if we want to add this CSS to a top-level selector. Maybe a class for the span would be a better option?

File: app/views/admin/facilities/index.html.erb [hunk 2]
```diff
@@ -42,18 +52,9 @@
...
-              <%= facility.state %>
-              <!-- <%= facility.country %></td-->
-              <%= facility.pin %>
-            </div>
-          </div>
-          <div class="col-md-3"><i class="fas fa-hospital light mr-2"></i> <%= facility.facility_type %> <span class="text-muted">(<%= facility.facility_size&.capitalize || "No size set" %>)</span></div>
+          <div class="col-md-1"><span class="text-muted"><%= facility.facility_type %></span></div>
+          <div class="col-md-2"><span class="text-muted"><%= facility.facility_size&.capitalize || "No size set" %></span></div>
+          <div class="col-md-2"><span class="text-muted">OPD: <%= facility.monthly_opd_load_was || "N/A" %></span></div>
         </div>
```
Comment (prabhanshuguptagit) [inline app/views/admin/facilities/index.html.erb]: Seems like a typo, should be `monthly_estimated_opd_load`.

File: app/views/admin/facilities/show.html.erb [hunk 3]
```diff
@@ -26,19 +26,25 @@
...
         <div><i class="fas fa-angle-left light"></i> None entered <i class="fas fa-angle-right light"></i></div>
       <% end %>
     </div>
   </div>
-  <div class="col-md-6">
+  <div class="col-md-4">
     <div class="card">
-      <h3>Facility Type &amp; Size</h3>
+      <h3>Estimated OPD load</h3>
+      <div><%= @facility.monthly_opd_load_was || "(N/A)" %> adult patients per month</div>
```
Comment (prabhanshuguptagit) [inline app/views/admin/facilities/show.html.erb]: Same, typo.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (harimohanraj89) [reply app/assets/stylesheets/application.scss]: Agree. @dburka what would you call these little heading spans?

File: app/views/admin/facilities/index.html.erb [hunk 4]
```diff
@@ -42,18 +52,9 @@
               <% end %>
             </h5>
           </div>
-          <div class="col-md-5">
-            <div class="card-details">
-              <i class="fas fa-map-marker-alt light mr-2"></i> <%= facility.street_address %>
-              <%= facility.village_or_colony %>
-              <%= facility.zone %>
-              <%= facility.district %>
```
Comment (prabhanshuguptagit) [inline app/views/admin/facilities/index.html.erb]: The address in the facilities list seems to be helpful in some cases https://simpledotorg.slack.com/archives/CK95QFJA2/p1580908551060100. @dburka is this good to get rid of?

Comment (harimohanraj89) [reply app/views/admin/facilities/index.html.erb]: Spoke to Daniel. We are good to axe 'em.

**Pull request merged**